scripts/tools/zap/zap_download.py

```diff
@@ -141,9 +141,9 @@ def _GetZapVersionToUse(project_root):
     # This heuristic may be bad at times, however then you can also override the
     # version in command line parameters
 
-    match_re = re.compile(r'.*ENV\s+ZAP_VERSION=([^# ]*)')
+    match_re = re.compile(r'.*ARG\s+ZAP_VERSION=([^# ]*)')
 
-    docker_path = os.path.join(project_root, "integrations/docker/images/chip-build/Dockerfile")
+    docker_path = os.path.join(project_root, "integrations/docker/images/chip-cert-bins/Dockerfile")
```

Comment:
Shouldn't this use `scripts/setup/zap.json`, which is what CI really uses?

Reply:
Cert-bins is tied to an older version (has a SHA hard-coded) so this is likely not the right place to get the version. zap.json is the correct place to take it from.